auth/oauth/types_mitzxnuts.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Nuts node
+ * Copyright (C) 2023 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+// Package oauth contains generic OAuth related functionality, variables and constants
+package oauth
+
+const (
+	// OpenIdCredIssuerWellKnown is the well-known base path for the openID credential issuer metadata as defined in OpenID4VCI specification
+	OpenIdCredIssuerWellKnown = "/.well-known/openid-credential-issuer"
+
+	OpenIdConfigurationWellKnown = "/.well-known/openid-configuration"
+)
+
+type OpenIDCredentialIssuerMetadata struct {
```

Comment:
As specified by RFC8141, this may be merged with AuthorizationServerMetadata. For code clarity a type alias can be defined.

Reply:
Alias created.